build.gradle

```diff
@@ -821,6 +824,7 @@ project('spring-xd-module') {
 		compile "org.springframework.boot:spring-boot-loader"
 		compile "com.fasterxml.jackson.core:jackson-databind"
 		compile project(':spring-xd-module-spi')
+		compile project(':spring-xd-spark-streaming')
```

Comment:
This feels wrong to me: why would we depend on spark stuff for the core module contract?